core-bundle/src/Resources/config/legacy_routing.yml

```diff
@@ -24,7 +24,7 @@ services:
             - '%contao.prepend_locale%'
             - '%contao.url_suffix%'
 
-    contao.routing.legacy_matcher:
+    contao.routing.matcher.legacy:
```

Comment:
I know that this would be the "correct" ID according to the linter, however, since there are no other services in the `contao.routing.matcher` namespace, I like `contao.routing.legacy_matcher` better. 🙈

Reply:
But we cannot normalize this, correct? Because not all services that are the only one in a namespace should be transformed that way I guess?

Reply:
This name is now suggested automatically 🎉